WowPacketParserModule.V7_0_3_22248/Parsers/MiscellaneousHandler.cs

```diff
@@ -333,5 +333,25 @@ public static void HandleOpenAlliedRaceDetailsGiver(Packet packet)
             packet.ReadPackedGuid128("GUID"); // Creature or GameObject
             packet.ReadInt32("RaceID");
         }
+
+        public static void ReadUnkAreaPoiUpdate(Packet packet, params object[] idx)
+        {
+            packet.ReadTime("UnkTime", idx);
+            packet.ReadInt32("UnkInt32_1", idx);    // AreaID?
+            packet.ReadInt32("UnkInt32_2", idx);
+            packet.ReadUInt32("UnkUInt32_3", idx);
+            packet.ReadUInt32("UnkUInt32_4", idx);
+        }
+
```

Comment:
packet.ReadTime("LastUpdate", idx);
packet.ReadInt32("QuestID", idx);    // AreaID?
packet.ReadInt32("Timer", idx);
packet.ReadUInt32("WorldStateVariableID", idx);
packet.ReadUInt32("WorldStateValue", idx);

Reply:
@funjoker

Reply:
it's not QuestID (too short numbers, just 4 diggits)